src/main/java/com/google/cloud/tools/io/FileUtil.java

```diff
@@ -28,23 +29,28 @@
 import java.nio.file.attribute.BasicFileAttributes;
 
 /**
- * Internal file utilities.
+ * File utilities we hope to eventually push up to Guava. 
```

Comment:
I'd love to talk about pushing this to Guava, but I'm not convinced it's valuable as a public javadoc on the class.